src/app/pages/charging-stations/charging-limit/charging-profile-limit/charging-station-charging-profile-limit.component.ts

```diff
@@ -44,10 +44,14 @@ export class ChargingStationChargingProfileLimitComponent implements OnInit, Aft
   @Input() public chargingStation!: ChargingStation;
 
   public profileTypeMap: ProfileType[] = [
-    { key: ChargingProfileKindType.ABSOLUTE, description: 'chargers.smart_charging.profile_types.absolute',
-      chargingProfileKindType: ChargingProfileKindType.ABSOLUTE, stackLevel: 3, profileId: 3 },
-    { key: ChargingProfileKindType.RECURRING, recurrencyKindType: RecurrencyKindType.DAILY, description: 'chargers.smart_charging.profile_types.recurring_daily',
-      chargingProfileKindType: ChargingProfileKindType.RECURRING, stackLevel: 2, profileId: 2 },
+    {
+      key: ChargingProfileKindType.ABSOLUTE, description: 'chargers.smart_charging.profile_types.absolute',
+      chargingProfileKindType: ChargingProfileKindType.ABSOLUTE, stackLevel: 3, profileId: 3
+    },
+    {
+      key: ChargingProfileKindType.RECURRING, recurrencyKindType: RecurrencyKindType.DAILY, description: 'chargers.smart_charging.profile_types.recurring_daily',
+      chargingProfileKindType: ChargingProfileKindType.RECURRING, stackLevel: 2, profileId: 2
```

Comment:
Same as above

Reply:
I do not think, that it is going to be static in the future.